### PR TITLE
🧹  Sonar smells

### PR DESCRIPTION
**Pull request by aldebaran30701**
### Prerequisites

- [X] Reviewed the [checklist](CHECKLIST.md)

- [ ] Reviewed feedback from the "Sonar Cloud" bot. Note that you have to wait
    for the "CI / Unit Tests") to complete first.

### Description of the Change
**Worked on the following rules:**
Add a private constructor to hide implicit public
Standard outputs shouldn't be used
Unused private fields should be removed
Rename fields to match regular expression
Constructors of abstract classes should be protected

### Applicable Issues
#1022

### CODE REVIEW

File: CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/infomap/flow/Network.java [hunk 1]
```diff
@@ -117,13 +117,11 @@ public void read() {
 
         Logf.printf("done! Found %d nodes and %d connections. ", rg.getVertexCount(), connectionMap.size());
         if (numDoubleLinks > 0) {
-            Logf.printf("%d connections was aggregated to existing connections. ", numDoubleLinks);
+            Logf.printf("%d connections was aggregated to existing connections. %n", numDoubleLinks);
```
Comment (antares1470) [inline CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/infomap/flow/Network.java:120]: Just noting you changed from `Logf` to `LOGGER` in a previous file but here you haven't. Not sure if this needs to change to using the java logger?

Comment (aldebaran30701) [reply CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/infomap/flow/Network.java:120]: I'll change those ones back to logF. I think it was early on in the branch and decided to go against changing them. I switched logf to use the logger either way.

Comment (antares1470) [reply CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/infomap/flow/Network.java:120]: It looks like you've changed the `Logf` to use the java logger over sysout so I guess that makes sense. That being said, I think that class is only created to be a wrapper around sysout so potentially that class could be removed all together and then you can just use the logger of the class that function was being called in

**Review by Delphinus8821 — APPROVED** [review]

File: CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/infomap/flow/Network.java (hunk 1 above)
Comment (aldebaran30701) [reply CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/infomap/flow/Network.java]: Removed all logF usages and replaced with logger.

**Review by antares1470 — APPROVED** [review]

**Pull request merged**